plugins/modules/helm.py

```diff
@@ -453,6 +459,9 @@ def deploy(
             yaml.dump(release_values, yaml_file, default_flow_style=False)
         deploy_command += " -f=" + path
 
+    if post_renderer:
+        deploy_command = " --post-renderer=" + post_renderer
```

Comment:
@mnaser Do you think we should add a logic to see if `post_renderer` binary exists or not? Or helm command takes care of that?

Reply:
It's best to let Helm take care of it:

```
❯ helm install --post-renderer=foo
Error: invalid argument "foo" for "--post-renderer" flag: unable to find binary at foo: exec: "foo": executable file not found in $PATH
```

Reply:
Great.